modules/varnish/manifests/monitoring.pp

```diff
@@ -29,11 +31,20 @@
   }
 
   $app_domain = hiera('app_domain')
-  $kibana_url = "https://kibana.${app_domain}"
   $hostname_prefix = split($::hostname, '-')
-  $kibana_search = {
-    'query' => "@source_host:${hostname_prefix[0]}* AND status:[500 TO 599]",
-    'from'  => '4h',
+
+  if $logit_kibana {
+    $kibana_url = 'https://logit.kibana.io'
```

Comment:
You've got kibana and logit the wrong way around here.

Reply:
Good catch!